### DISCUSSION TITLE
How to POST binary file

### DISCUSSION THREAD

**Discussion by ramisedhom** [usebruno/bruno#1283] [Q&A] [answered]
I can't find  the option to attach binary file in the Body of a POST request. Thanks to advice how you use Bruno to post binary files.

**Comment by rsilvr** [comment, marked as answer]
It seems it's not supported yet, but there are some workarounds:
- https://github.com/usebruno/bruno/discussions/385#discussioncomment-7196674
- https://github.com/usebruno/bruno/discussions/385#discussioncomment-7233816

**Comment by fseidl-bauradar** [comment]
Is there still no solution to add files using postman other than by script
